galasa-parent/dev.galasa.framework.api.ras/src/main/java/dev/galasa/framework/api/ras/internal/TestClassesRas.java

```diff
@@ -43,6 +43,8 @@ public class TestClassesRas extends HttpServlet {
 	 */
 	private static final long serialVersionUID = 1L;
 
+	private SortQueryParameterChecker SortQueryParameterChecker;
```

Comment:
The 2nd one should start with lower-case letter for variables.

Reply:
You could also say
```
private SortQueryParameterChecker sortQueryParameterChecker = new SortQueryParameterChecker();
```
to avoid a null pointer exception.